ionoscloud/resource_ipfailover.go

```diff
@@ -53,28 +61,31 @@ func resourceLanIPFailover() *schema.Resource {
 
 func resourceLanIPFailoverCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
 	client := meta.(services.SdkBundle).CloudApiClient
-	dcid := d.Get("datacenter_id").(string)
-	lanid := d.Get("lan_id").(string)
-	if lanid == "" {
-		diags := diag.FromErr(fmt.Errorf("'lan_id' is missing, please provide a valid lan ID "))
-		return diags
-	}
+	dcId := d.Get("datacenter_id").(string)
+	lanId := d.Get("lan_id").(string)
 	ip := d.Get("ip").(string)
 	nicUuid := d.Get("nicuuid").(string)
 
-	properties := &ionoscloud.LanProperties{}
+	// First, retrieve the existent IP Failover groups
```

Comment:
`existent` to `existing`

Reply:
@cristiGuranIonos done in 7a35821c28be88119348db39cb88c96db7c113b9